src/app/(admin)/new_dashboard/guides/components/SearchBar/index.tsx

```diff
@@ -42,6 +60,16 @@ const SearchBar = ({ searchQuery, onSearchChange, selectedTag, onTagChange, cate
         getOptionLabel={(option) => option}
       />
     </FormControl>
+    <FormControl variant="outlined" sx={{ ml: 2, minWidth: 150 }}>
+      <Select
+        value={sortCriteria}
+        onChange={handleSortChange}
+        displayEmpty
+      >
+        <MenuItem value="rating">Sort by Rating</MenuItem>
```

Comment:
again, what happens if we want to add another category? 
it is better to make an array of possible values and map them to menuItem:
possible psuodo code example:
`
["rating" , "date"].map(category =>
<<MenuItem value={category}>Sort by Rating</MenuItem>
`

Reply:
ok done